docs/userguides/config.md

```diff
@@ -3,6 +3,22 @@
 An `ape-config.yaml` file allows you to configure ape. This guide serves as an index of the settings you can include 
 in an `ape-config.yaml` file.
 
+## Contracts Folder
+
+Specify a different path to your `contracts/` directory.
+This is useful when using a different naming convention, such as `src/` rather than `contracts/`.
+
+```yaml
+contracts_folder: src
+```
+
+You can also use an absolute path.
+This is useful for projects that compile contracts outside their directory.
+
+```yaml
+contracts_folder: "~/GlobalContracts"
```

Comment:
Not sure I've seen a use case for this (global contracts dir), although it is probably very useful for referencing other projects locally as well

Reply:
Yeah, you can quickly "extend" a project this way. Maybe for like a set of experimental or custom scripts you have.

Reply:
Sure! I don't see why not to allow it, just have to see what people do with this feature lol